src/aura/CRLP_RollupsContainer/CRLP_RollupsContainerHelper.js

```diff
@@ -1,7 +1,7 @@
 ({
     /* @description: resets the view assignments, clears detail information, and displays rollup grid
```

Comment:
Is it worth fixing all the apexDoc comments in the JS to move the `@description` line down below `/**`?

Reply:
I think so! I've been moving them down gradually, but this is something we can do when we remove the console.logs (if not before).